bundles/binding/org.openhab.binding.pilight/src/main/java/org/openhab/binding/pilight/internal/PilightBinding.java

```diff
@@ -247,9 +206,9 @@ private PilightBindingProvider findFirstMatchingBindingProvider(String itemName)
 		return null;
 	}
 	
-	private List<PilightBindingConfig> getConfigs(String instance, String location, String device) {
+	private List<PilightBindingConfig> getConfig(String instance, String device) {
```

Comment:
why did you chance the name to singular? A list is returned here so the method name should reflect that properly